### PR TITLE
Refactored tool stack section.

### PR DESCRIPTION
**Pull request by samoht9277**
* Added a link to a Notion document explaining what tools LambdaClass use in a daily basis.
* Removed Slack section as it is already on the new Notion document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,16 +258,33 @@ Use git and commit often, even in one-person projects.
...
+    - _Gmail_ is used by your `lambdaclass.com` account to manage e-mails.
+    - _Meets_ is used for video conferences. It's usually used for broadcasting information to all employees.
+- GitHub
+  - We use **GitHub** to store all our code! This platform let developers contribute to a centralized project, allowing synchronous progress for the project.
+- Kandji
+  - **Kandji** manages our MacBook fleet. We mainly enforce some security standards across all Macs.
+- Tailscale
+  - **Tailscale** manages and controls access to our servers. Think of it as our own VPN. Authorization is done via **Duo**, which proves your identity. With Tailscale, SSH public keys are deprecated.
+- Notion
+  - **Notion** helps to document any information worth saving and sharing to the rest of the company.
```
Comment (Loco1982) [inline README.md:287]:
```suggestion
  - **Notion** helps to document any information worth saving and sharing with your team or the rest of the company.
```

File: README.md [hunk 2]
```diff
@@ -258,16 +258,33 @@ Use git and commit often, even in one-person projects.
...
+	2. Feel free to join any public channel that sparks your interest *but* be careful with your messages on team-related channels, especially if you don't belong to that team.
+	3. We encourage everyone to join in the discussions in Slack, don't be shy if you think you have something interesting to say about any topic.
+	4. Try to always post according to the topic of the channel. Straying a little from it is not an issue but if you want to joke about something completely unrelated to the channel please post it on the #random channel.
+	5. As a rule of thumb most of your Slack communication has to happen on the channels and not on the DMs.
+   		1. This last one is particularly relevant when talking with someone outside of Lambda, always defer to channels when talking with them.
+	6. Try to not create channels unless they're really necessary. The idea is to have only the channels that we need, please don't create channels to make some kind of joke.
+- Duo
+  - **Duo** is our 2FA way of validating your identity. Tasks such as logging into Slack, 1Password or GitHub require you to prove your identity, besides your password.
```
Comment (Loco1982) [inline README.md:273]:
```suggestion
  - **Duo** is our 2FA way of validating your identity. Tasks such as logging into Slack, 1Password, or GitHub require you to prove your identity, besides your password. 
```

File: README.md [hunk 3]
```diff
@@ -258,16 +258,33 @@ Use git and commit often, even in one-person projects.
...
+	4. Try to always post according to the topic of the channel. Straying a little from it is not an issue but if you want to joke about something completely unrelated to the channel please post it on the #random channel.
+	5. As a rule of thumb most of your Slack communication has to happen on the channels and not on the DMs.
+   		1. This last one is particularly relevant when talking with someone outside of Lambda, always defer to channels when talking with them.
+	6. Try to not create channels unless they're really necessary. The idea is to have only the channels that we need, please don't create channels to make some kind of joke.
+- Duo
+  - **Duo** is our 2FA way of validating your identity. Tasks such as logging into Slack, 1Password or GitHub require you to prove your identity, besides your password. 
+- 1Password
+  -  We use **1Password** to store all our secrets and shared passwords. Most projects have a thing called _Vault_, which is a location where you can share secrets and passwords with your team. A personal suscription is included for you to store your own passwords. The company cannot see nor edit your private account.
```
Comment (Loco1982) [inline README.md:275]:
```suggestion
  -  We use **1Password** to store all our secrets and shared passwords. Most projects have a thing called _Vault_, which is a location where you can share secrets and passwords with your team. A personal subscription is included for you to store your own passwords. The company cannot see nor edit your private account.
```

File: README.md [hunk 4]
```diff
@@ -258,16 +258,33 @@ Use git and commit often, even in one-person projects.
...
+- Google
+  - **Google** is used to manage multiple things related to communication.
+    - _Gmail_ is used by your `lambdaclass.com` account to manage e-mails.
+    - _Meets_ is used for video conferences. It's usually used for broadcasting information to all employees.
+- GitHub
+  - We use **GitHub** to store all our code! This platform let developers contribute to a centralized project, allowing synchronous progress for the project.
```
Comment (Loco1982) [inline README.md:281]:
```suggestion
  - We use **GitHub** to store all our code! This platform lets developers contribute to a centralized project, allowing synchronous progress for the project.
```

**Review by Loco1982 — CHANGES_REQUESTED** [review]
Don't forget to put a description to your PR. No PR should be merged if they don't have a description. Thanks!

File: README.md (hunk 4 above)
Comment (samoht9277) [reply README.md:281]: Nice catches!

**Review by Loco1982 — APPROVED** [review]
LGTM

**Review by tomasdema — APPROVED** [review]
lgtm

**Review by fmoletta — APPROVED** [review]
🚀

**Review by klaus993 — APPROVED** [review]

**Review by Loco1982 — APPROVED** [review]
LGTM

**Pull request merged**